<launch>

  <arg name="UAV_NAME" default="$(env UAV_NAME)"/>
  <arg name="custom_config" default="" />

  <group ns="$(arg UAV_NAME)">

    <node name="RandomFlier" pkg="nodelet" type="nodelet" args="standalone mrs_uav_testing/RandomFlier" output="screen" >

      <rosparam file="$(find mrs_uav_testing)/config/random_flier.yaml" />

      <!-- Load a user param file -->
      <rosparam if="$(eval not arg('custom_config') == '')" file="$(arg custom_config)" />

      <param name="world_frame" value="$(arg UAV_NAME)/local_origin" />

        <!-- Subscribers -->
      <remap from="~position_command_in" to="control_manager/tracker_cmd" />

        <!-- Publishers -->
      <remap from="~reference_out" to="control_manager/reference" />

        <!-- Services -->
      <remap from="~activate_in" to="automatic_start/start" />

    </node>

  </group>
</launch>
